### DISCUSSION TITLE
How to position item in center and right with flex?

### DISCUSSION THREAD

**Discussion by BearCooder** [tailwindlabs/tailwindcss#12450] [Help] [answered]
Hi,

this may seem like a silly question but I am kind of stuck.
I have two elements. And I want to position the first element in the center and the second element on the right just like this:
![grafik](https://github.com/tailwindlabs/tailwindcss/assets/104575503/3faafe31-96ab-4512-a881-36caddb761f9)

Right now I achieve this effect by having three element, where the first one is an empty div and my parent element uses justify between.
```
<div class="mx-2 flex justify-between p-2 lg:p-5">
  <div></div>
  <div>Center</div>
  <div>Right</div>
</div>
```
But I know this is just not ideal and hacky. How can I achieve it without the empty div?
I already tried so many ways, by trying to override the main behaviour of my divs with `justify-center` and the other one with` justify-end` but it does not change the outcome. 
This is the tailwind playground: 
https://play.tailwindcss.com/C1yn3fkAsj

I appreciate any help.

**Comment by wongjn** [comment]
You could consider using a grid layout [like so](https://play.tailwindcss.com/sGM1qvx3wE).

**Comment by makibat** [comment, marked as answer]
Hi @BearCooder, working off the playground you've provided, just adding [flex-1](https://play.tailwindcss.com/8OqDuI9SBH) to "Center" should do the trick:

```js
<div class="mx-2 flex items-end justify-center p-2 lg:p-5">
  <div class="flex flex-1 justify-center">Center</div>
  <div class="flex justify-end">Right</div>
</div>
```

Hope that helps!

**Comment by BearCooder** [comment]
Amazing both answers are perfect thank you!

**Comment by jcsmileyjr** [comment]
Great answer
